style.css

```diff
@@ -1,10 +1,113 @@
-body {
-background-color: grey;
-font-family: "GothamPro";
+ * {
+  margin: 0;
+  padding: 0;
+  font-family: 'Gotham Pro';
+  background-color: grey;
 }
-
-.item__price {
-font-weight: bold;
+.body {
+  box-sizing: border-box;
+}
+.container {
+  max-width: 1030px;
+  margin: 0 auto;
+}
+/*header*/
+.header {
+  max-height: 96px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+.header-menu,
+.header-menu__right {
+  display: flex;
+  margin-top: 28px;
+  justify-content: space-between;
+}
+.header-menu__logo {
+  margin-top: 32px;
+  margin-left: 35px;
+ }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step/pullRequest?prid=5930829)